packages/sources/blockchair/src/endpoint/balance.ts

```diff
@@ -21,10 +21,13 @@ const getBalances: balance.GetBalances = async (accounts, config) => {
 
   const reqConfig: RequestConfig = {
     ...config.api,
-    url: getBalanceURI(addresses, coin as string, chain as string),
+    url: getBalanceURI(coin as string, chain as string),
```

Comment:
Are these `... as string` necessary? Do we not have types covered for them?

Reply:
`coin` and `chain` come from the `Account` type, which defines them as optional, so if we don't cast them, they cause a typescript error. I could update this method to check if they're valid strings before doing anything else if you think this is an issue

Reply:
Hmm yeah I think we should do that. If they're not required, I guess we could just do `... || ''` instead?

Reply:
+1 for `... || ''`, or `... ?? ''` if the type is limited to `string | undefined | null`